src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="POST" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="form-container">
+        <legend>Personal information</legend>
+        <label for="surname" class="input-label">Surname:
+          <input 
+            type="text" 
+            id="surname" 
+            name="surname" 
+            autocomplete="off" 
+            required
+          >
+        </label>
+        <label for="name" class="input-label">Name:
+          <input 
+          type="text" 
+          id="name" 
+          name="name" 
+          autocomplete="off" 
+          required
+          >
+        </label>
+        <label for="age" class="input-label">How old are You?
+          <input 
+            type="number" 
+            id="age" 
+            name="age" 
+            min="1" 
+            max="100" 
+            value="12" 
+            autocomplete="off" 
+            required
+          >
+        </label>
+        <label for="dob" class="input-label">Full date of birth:
+          <input 
+          type="date" 
+          id="dob" 
+          name="dob" 
+          autocomplete="off" 
+          required
```

Comment:
you didn't fix all indentation problems
```suggestion
          <input 
             type="date" 
            id="dob" 
            name="dob" 
            autocomplete="off" 
            required
```